pipe.sh

```diff
@@ -15,6 +15,11 @@ composer_setup () {
       composer create-project --repository-url="$REPOSITORY_URL" "$MAGENTO_VERSION" /magento2 --no-install
       cd /magento2
       composer config repositories.local path $BITBUCKET_CLONE_DIR
+      if [ -z "${COMPOSER_PACKAGES}" ]; then
+          echo "COMPOSER_PACKAGES environment variable is not set - aborting"
+          exit 1
```

Comment:
if it didn't exist and `composer require` was moved in here, we would be able to use this on both standalone modules and a whole project with modules. or is the intention to only work with standalone module?

Reply:
For whole projects, `composer.lock` should already be present, so it won't get here